xbmc/filesystem/NFSDirectory.h

```diff
@@ -11,6 +11,11 @@
 #include "IDirectory.h"
 #include "NFSFile.h"
 
+#include <memory>
+#include <vector>
+
+class CFileItem;
+class CFileItemList;
```

Comment:
There is this method defined in the header:

```c++
CacheType GetCacheType(const CURL& url) const override { return CacheType::ONCE; }
```

 For `CacheType::ONCE` you need to include `CFileitemList.h`, where this value is defined. 
 Alternatively, you could outline the implementation.

Reply:
CacheType::ONCE seems defined in IDirectory.h

https://github.com/xbmc/xbmc/blob/ebd6ba841b25e057888afa0dcc2f78314b729e29/xbmc/filesystem/IDirectory.h#L28-L33

Reply:
Oh nice. Was on the phone. Sorry for the noise. All good then.